generators/app/tasks/appSetup/index.js

```diff
@@ -50,18 +53,28 @@ module.exports = function index() {
   }
 
   // ----------------     features     ----------------
-  if (this.features.pushnotifications) {
-    pushNotificationsFeatureFiles.bind(this)();
+  if (this.features.crashlytics || this.features.googleanalytics || this.features.pushnotifications) {
+    firebaseCoreFeatureFiles.bind(this)();
   }
+
   if (this.features.login) {
     loginFeatureFiles.bind(this)();
   }
   if (this.features.googleanalytics) {
     googleAnalyticsFeatureFiles.bind(this)();
   }
 
+  if (this.features.pushnotifications) {
+    pushNotificationsFeatureFiles.bind(this)();
+  }
+
   // --------------- Enables fullscreen on iPad ----------------------------
   enableFullscreen.bind(this)();
 
+  // ----------------     Install pod with added code for setting xcodeproj     ----------------
+  if (this.features.crashlytics || this.features.googleanalytics || this.features.pushnotifications) {
+    installPods.bind(this)();
```

Comment:
since it's the same `if` statement, we could unify them and add the text above as a line comment

Reply:
This is placed at the end because in crashlytics we add code in the podfile, maybe in analytics or push too.